tests/ConfigurationException/InvalidConfigurationExceptionTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\ConfigurationException;
+
+use PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException;
+use PhpCsFixer\Console\Command\FixCommand;
+use PHPUnit\Framework;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException
+ */
+final class InvalidConfigurationExceptionTest extends Framework\TestCase
```

Comment:
@SpacePossum except of this particular case...
if not, our Travis would crash on lowest deps ;)